frontend/src/components/MapView/MapTooltip/PopupCharts/PopupAnalysisCharts.tsx

```diff
@@ -73,8 +76,8 @@ const PopupAnalysisCharts = ({
     levels: item.chartData?.levels,
   }));
 
-  const startDate = new Date().getTime() - oneYearInMs;
-  const endDate = new Date().getTime();
+  const { startDate: endDate } = useSelector(dateRangeSelector);
```

Comment:
This question is not about this PR. Why this props is called startDate by default and not endDate or selectedDate? Do you know what are the cases where this cursor is refering a startDate?
![image](https://github.com/WFP-VAM/prism-app/assets/100352902/5854bea8-0e87-4c46-998f-1707e82f7455)

Reply:
The dateRangeSelector has a startDate and an endDate, but at the moment it seems that we are only using the startDate. I will open an issue to track the potential for simplification.